lib/components/dropdown.vue

```diff
@@ -43,15 +43,6 @@
     </div>
 </template>
 
-<style scoped>
-    .dropdown-item:focus,
-    .dropdown-item:hover,
-    .dropdown-header:focus {
-        background-color: #eaeaea;
```

Comment:
![image](https://user-images.githubusercontent.com/5158436/29223653-924e7c9a-7edc-11e7-8d61-e592e886468a.png)

(the former bug when mouse down)